### ISSUE TITLE
Model collapse

### ISSUE THREAD

**Issue by dance-ing** [akanimax/BMSG-GAN#38]
Why are all the images generated by me using the trained network exactly the same?
I observed the training process, and the images generated using different noises are still somewhat different, but gradually they become the same image. What is the reason? Is my training data too small?

**Comment by nguyenpbui** [comment]
I faced the same problem even trying to change the GAN structure, add batch normalization,..
Did you find the answer yet?

**Comment by dance-ing** [comment]
sorry, not yet...................

**Comment by nihadguluzade** [comment]
I'm facing the same issue while trying to train my own dataset. What is the size of your dataset?

**Comment by nguyenpbui** [comment]
400 images from LFW dataset. But I'm using celaba dataset now.

**Comment by akanimax** [comment]
Hi guys!
imho, 400 images is an extremely small dataset and mode collapse is expected. I'd suggest trying the data - augmentation suggested in [https://arxiv.org/abs/2006.06676](https://arxiv.org/abs/2006.06676) rather than changing the network architecture. There also a very informative video about the paper [here](https://arxiv.org/abs/2006.06676).
Hope this helps!

Cheers :beers:!
